tests/test_ranges.py

```diff
@@ -996,6 +996,128 @@ def test_copies(self, obj):
         assert obj == copy.copy(obj) == copy.deepcopy(obj)
 
 
+class TestIterateOverMonths:
+    @pytest.mark.parametrize(
+        "row",
+        [
+            {
+                "period": ranges.FiniteDatetimeRange(
```

Comment:
:panda_face: could we have ids for these tests?

Reply:
I've added ids in 768be2148b8c65b218bc0da540efcdfaba57bf45